aplt/runner.py

```diff
@@ -114,39 +116,181 @@ def remove_client(self, ws_client):
                 connectWS(self._factory, contextFactory=self._factory_context)
             return
 
-    def remove_processor(self, processor):
+    def remove_processor(self):
         """Remove a completed processor"""
-        self._processors.pop(processor, None)
+        self._processors -= 1
+
+
+class LoadRunner(object):
+        """Runs a bunch of scenarios for a load-test"""
+        def __init__(self, scenario_list):
+            """Initializes a LoadRunner
+
+            Takes a list of tuples indicating scenario to run, quantity,
+            stagger delay, and overall delay.
```

Comment:
Yeah, I get that, https://github.com/mozilla-services/ap-loadtester/pull/7/files#diff-3f17b35054774013f16876372d4c0f7dR256 shows how the function is fed from the test-plans. You might want to add a check for the number of args so that folks don't accidentally miss one. (e.g. they drop quantity or something). I was more thinking that the test plan might look like 
`aplt_testplan wss://example.com {"scenario-funciton": "foo:bar", "quantity": 100, "stagger": 50, "delay": 10, "args": {}}`
or something. but that raises it's own set of issues.